os/net/netmgr/netmgr_logger.c

```diff
@@ -0,0 +1,86 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <tinyara/net/netlog.h>
+#include <tinyara/kmalloc.h>
+
+#define NETLOGGER_BUF_SIZE 256
+
+int netlogger_init(netmgr_logger_p *log)
+{
+	netmgr_logger *tlog = (netmgr_logger *)malloc(sizeof(netmgr_logger));
```

Comment:
I think it is better to use `kmm_malloc` instead of `malloc`.
Because this is kernel side, not user side.